client/object_put.go

```diff
@@ -261,7 +261,7 @@ func (x *ObjectWriter) Close() (*ResObjectPut, error) {
 //
 // Returns errors:
 //   - [ErrMissingSigner]
-func (c *Client) ObjectPutInit(ctx context.Context, hdr object.Object, signer neofscrypto.Signer, prm PrmObjectPutInit) (*ObjectWriter, error) {
+func (c *Client) ObjectPutInit(ctx context.Context, hdr object.Object, signer user.Signer, prm PrmObjectPutInit) (*ObjectWriter, error) {
```

Comment:
Not needed.

Reply:
The same like [here](https://github.com/nspcc-dev/neofs-sdk-go/pull/475#discussion_r1274405118)